src/components/subscriptions/licenses/LicenseManagementModals/tests/LicenseManagementRevokeModal.test.jsx

```diff
@@ -127,6 +127,111 @@ describe('<LicenseManagementRevokeModal />', () => {
+
+        expect(onSuccessMock).not.toBeCalled();
+        expect(logError).toBeCalledTimes(1);
+      });
+
+      it('handles 207 partial success with only 404 errors correctly', async () => {
+        const mockPartialSuccess207WithOnly404 = {
+          status: 207,
+          data: {
+            error_messages: [{ error_response_status: 404 }],
+          },
+        };
+        LicenseManagerApiService.licenseBulkRevoke.mockResolvedValue(mockPartialSuccess207WithOnly404);
+
+        render(<LicenseManagementRevokeModal {...props} />);
+        await act(async () => { userEvent.click(screen.getByText('Revoke (1)')); });
+
+        expect(onSuccessMock).toBeCalledTimes(1);
+        expect(logError).not.toBeCalled();
+      });
+
+      it('handles 207 partial success with mixed errors correctly', async () => {
+        const mockPartialSuccess207WithMixedErrors = {
+          status: 207,
+          data: {
+            error_messages: [
+              { error_response_status: 404 },
+              { error_response_status: 400 },
+            ],
+            revocation_results: [
+              { license_uuid: 'license-uuid-1', original_status: 'assigned', user_email: 'user1@example.com' },
+              { license_uuid: 'license-uuid-2', original_status: 'activated', user_email: 'user2@example.com' },
+            ],
+          },
+        };
+        LicenseManagerApiService.licenseBulkRevoke.mockResolvedValue(mockPartialSuccess207WithMixedErrors);
+
+        render(<LicenseManagementRevokeModal {...props} />);
+        await act(async () => { userEvent.click(screen.getByText('Revoke (1)')); });
+
+        expect(onSuccessMock).not.toBeCalled();
+        expect(logError).toBeCalledTimes(1);
+      });
+
+      it('handles 207 partial success with 404 errors and successful revocations correctly', async () => {
+        const mockPartialSuccess207WithMixed404AndSuccess = {
+          status: 207,
+          data: {
+            error_messages: [
+              { error_response_status: 404, user_email: 'user1@example.com' },
+              { error_response_status: 404, user_email: 'user2@example.com' },
+            ],
+            revocation_results: [
+              { license_uuid: 'license-uuid-3', original_status: 'assigned', user_email: 'user3@example.com' },
+              { license_uuid: 'license-uuid-4', original_status: 'activated', user_email: 'user4@example.com' },
+            ],
+          },
+        };
+        LicenseManagerApiService.licenseBulkRevoke.mockResolvedValue(mockPartialSuccess207WithMixed404AndSuccess);
+
+        render(<LicenseManagementRevokeModal {...props} />);
+        await act(async () => { userEvent.click(screen.getByText('Revoke (1)')); });
+
+        expect(onSuccessMock).toBeCalledTimes(1);
+        expect(onSuccessMock).toHaveBeenCalledWith(mockPartialSuccess207WithMixed404AndSuccess.data);
+        expect(logError).not.toBeCalled();
+      });
+    });
```

Comment:
[gratitude] Thank you for adding unit tests to this already existing functionality! 🙌